packages/expo-maps/src/google/GoogleMapsView.tsx

```diff
@@ -56,6 +58,12 @@ export const GoogleMapsView = React.forwardRef<GoogleMapsViewType, GoogleMapsVie
     const onNativePOIClick = useNativeEvent(onPOIClick);
     const onNativeMarkerClick = useNativeEvent(onMarkerClick);
     const onNativeCameraMove = useNativeEvent(onCameraMove);
+    const onNativePolylineClick = useNativeEvent(onPolylineClick);
+
+    const parsedPolylines = polylines?.map((polyline) => ({
+      ...polyline,
+      color: polyline.color ? processColor(polyline.color) : undefined,
```

Comment:
```suggestion
      color: processColor(polyline.color),
```
I don't think that check is necessary.

Reply:
I'd love to remove it, but at the very least what we need is `color: processColor(polyline.color) ?? undefined` since processColor can return null, which causes typescript to complain.